ext/webextension/src/browser_action/main_popup.js

```diff
@@ -524,17 +524,16 @@ function indetifierUpdate() {
             else
                 a += text.charCodeAt(i);
         }
-        a = a % 90;
-        b = b % 90;
-        if (a < 12) 
-            a += 12;
-        if (b < 12)
-            b += 12;
-
-        // transponse from entity coding to codePoint
-        a -= 12;
-        b -= 12;
-        identifier.textContent = String.fromCodePoint(emojiOffset + a, emojiOffset + b);
+
+
+	// emoji range between 12 - 90
+        a = a % 78;
```

Comment:
Hi I'm not sure this turned out any better 

If we look here: https://apps.timwhitlock.info/emoji/tables/unicode (section 1 and 6a) We can see that the set you are targeting are from codepoints 1f600 to 1f64f,  80 (0x50) symbols - why do you want to use 78? 

Also, when I changed the code to use String.fromCodePoint (instead of innerHTML and html entity encoding) the +12 offset is no longer needed. To keep your calculations and avoid changing original behaviour of your code, I did the -12 at the end (#128512  = unicode code point 1f600)

Now, if we accept different behaviour (by changing the modulo), why not just 
```
a %= 0x50
b %= 0x50
identifier.textContent = String.fromCodePoint(emojiOffset + a, 32, emojiOffset + b);
```

Another consideration is this note "The additional sections refer to symbols that have no mapping to Japanese mobile carriers."   I'm not entirely sure how to interpret that, but it does sound like the 6a symbols are not available in Japanese locale?